lib/flood_risk_engine/engine.rb

```diff
@@ -12,12 +12,32 @@ class Engine < ::Rails::Engine
       app.config.i18n.load_path += Dir[config.root.join("config/locales/**/**/", "*.{rb,yml}").to_s]
     end
 
+    # Automatically add our migrations into the main apps migrations
+    initializer :append_migrations, before: :load_config_initializers do |app|
+      unless app.root.to_s.match root.to_s
+        config.paths["db/migrate"].expanded.each do |expanded_path|
+          app.config.paths["db/migrate"] << expanded_path
+        end
+      end
+    end
+
```

Comment:
this was you do not need to run a separate rake task. db:migrate just pulls them in ?